modules/runners/lambdas/runners/src/simple-pool/simple-pool.ts

```diff
@@ -0,0 +1,91 @@
+import yn from 'yn';
+import { logger as rootLogger } from './../logger';
+import { createRunners } from '../scale-runners/scale-up';
+import { createGithubAppAuth, createGithubInstallationAuth, createOctoClient } from '../gh-auth/gh-auth';
+import { listEC2Runners } from '../aws/runners';
+
+const logger = rootLogger.getChildLogger({ name: 'simple-pool' });
+
+export interface SimplePoolEvent {
+  simplePoolSize: number;
+}
+
+export async function adjust(event: SimplePoolEvent): Promise<void> {
+  logger.info(`Adjusting simple pool of size: ${event.simplePoolSize}`);
```

Comment:
I don't think we are adjusting anything at this point.

```suggestion
  logger.info(`Checking current pool size against simple pool of size: ${event.simplePoolSize}`);
```